app/src/main/java/com/example/pocketcloset/LoginActivity.java

```diff
@@ -32,7 +29,7 @@ protected void onCreate(Bundle savedInstanceState) {
             goMainActivity();
```

Comment:
You should also  probably `return` here. There's no reason to do additional setup after this, and continuing could cause bugs in the future